argocd/overlays/moc-infra/applications/envs/osc/osc-cl2/cluster-management/kfp-tekton.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kfp-tekton
+spec:
+  destination:
+    name: osc-cl2
+    namespace: kubeflow
+  project: operate-first
```

Comment:
Are we sure the project should be `operate-first`? all the other argocd cluster resource applications in this overlay use `os-climate`.